torch/_dynamo/allowed_functions.py

```diff
@@ -179,6 +210,19 @@ def _is_allowed_module_prefix(obj):
             # issues observed in
             # https://github.com/pytorch/pytorch/issues/108269
             "torch.distributed.algorithms.",
+            "torch._custom_ops",
+            "torch._dynamo",
+            "torch._functorch.aot_autograd",
+            "torch._functorch.compile_utils",
+            "torch._functorch.partitioners",
+            "torch._functorch.python_key",
+            "torch._higher_order_ops.cond",
+            "torch.ao.quantization.backend_config",
+            "torch.ao.quantization.fx._",
+            "torch.ao.quantization.quantizer",
+            "torch.backends.xnnpack",
+            "torch.jit.trace",
+            "torch.ops.foo",
```

Comment:
Is this list being copied some somewhere else?  Where did it come from?

Reply:
I guess this is something @Kronuz manually added to make CI happy, please correct me if I'm wrong. I don't think we should do something like this. The major change in this PR is lazily generating the allow function list, however, because of some side effect of torch module, which makes the eager allow function list (original one) is different from the list generated by lazy import (change in this PR). I'm planning to refactor this as a pre-generated (or hard code ) function list like ```native_functions.yaml```, then we don't need to generating the list when initializing dynamo.